www/layouts/newsletter/section.html

```diff
@@ -0,0 +1,465 @@
+{{ define "main" }} {{ block "header" . }} {{ partialCached "header" . }}
+{{end}}
+<div id="newsletter-main-section" class="page-single">
+  <div class="px-lg-5 mt-6">
+    <div class="row">
+      <div class="col-12">
+        <h1>Subscribe to our newsletter</h1>
+      </div>
+    </div>
+    <div class="row mt-5">
+      <div class="col-12">
+        <p>
+          Subscribe to the free monthly "MIT OpenCourseWare Update"
+          e-newsletter.
+        </p>
+        <p>You'll receive notification of OCW courses, news, and marketing.</p>
+      </div>
+    </div>
+  </div>
+  <!-- Begin Mailchimp Signup Form -->
+  <link
+    href="//cdn-images.mailchimp.com/embedcode/classic-10_7.css"
+    rel="stylesheet"
+    type="text/css"
+  />
+  <style type="text/css">
+    #mc_embed_signup {
+      background: #fff;
+      clear: left;
+      font: 14px Helvetica, Arial, sans-serif;
+    }
+    /* Add your own Mailchimp form style overrides in your site stylesheet or in this style block.
```

Comment:
Should we move this `cdn link` to HEAD and `inline style` to css file? 
Like suggested in this mailchimp comment

Reply:
moved